christin/index.js

```diff
@@ -0,0 +1,182 @@
+//?Addition.Write a program to add up the numbers 1 to 20.
+
+for (let i = 0; i <= 20; i++) {
+  let sum = i + i;
+  console.log("1.", sum);
+}
+
+//0 + i = 1
+//1 + i = 3
+
+//?There are i bottles of beer on the wall.Write a program that will output, "There is one bottle of beer on the wall." "There are two bottles of beer on the wall" up until there are five bottles.
+
+for (let i = 0; i <= 5; i++) {
+  console.log(`2: There is ${i} bottle of beer on the wall.`);
+}
```

Comment:
2nd Task : We need if else condition to check if there was just 1 bottle then print me There is one bottle, otherewise(if we have more than one) print There ARE ..... bottles

Reply:
right, changed .